ecies/aes.go

```diff
@@ -0,0 +1,70 @@
+package ecies
```

Comment:
I am not always against copying utils directly into the main project, but is it possible to have this done in an external library and just import/wrap helpers as needed?